starboard/xb1/tools/xb1_launcher.py

```diff
@@ -448,6 +456,27 @@ def CheckPackageIsDeployed(self):
       return False
     return package_list[package_index:].split('\n')[0].strip()
 
+  def RunTest(self, appx_name: str):
+    self.net_args_thread = None
+    attempt_num = 0
+    retry_wait_s = _TEST_RETRY_WAIT
+    while attempt_num < _TEST_MAX_TRIES:
+      if not self.net_args_thread or not self.net_args_thread.is_alive():
+        # This thread must start before the app executes or else it is possible
+        # the app will hang at _network_api.ExecuteBinary()
+        self.net_args_thread = net_args.NetArgsThread(self.device_id,
+                                                      _XB1_NET_ARG_PORT,
+                                                      self._target_args)
+      if self._network_api.ExecuteBinary(_DEFAULT_PACKAGE_NAME, appx_name):
+        break
+
```

Comment:
is it worth logging or asserting that the net-args thread from before is still alive here?